tp_timesheet/__main__.py

```diff
@@ -64,6 +66,13 @@ def run():
         for date in dates:
             submit_timesheet(URL, EMAIL, date, args.debug)
 
+        # Notification (OSX only)
+        if args.notification and sys.platform.lower() == 'darwin':
+            notification_text = 'Timesheet for today is successfully submitted.'
+            if len(dates) != 1:
+                notification_text = 'Timesheets from {} to {} are successfully submitted.'.format(dates[0], dates[-1])
+            os.system("""osascript -e 'display notification "{}" with title "TP_timesheet"'""".format(notification_text))
```

Comment:
```suggestion
            os.system("""osascript -e 'display notification "{}" with title "TP Timesheet"'""".format(notification_text))
```
I think this looks cleaner for title